gcp/gcp.go

```diff
@@ -75,6 +76,29 @@ func DefaultCredentials(ctx context.Context) (*google.Credentials, error) {
 	return adc, nil
 }
 
+// FakeDefaultCredentials gets GCP credentials from a fake JSON credentials
+// file in memory. This function is meant for use as a helper in example code,
+// to be replaced by gcp.DefaultCredentials(ctx) or google.CredentialsFromJSON
+// in non-example code.
+// See https://cloud.google.com/docs/authentication/production
+// for more info on alternatives.
+func FakeDefaultCredentials(ctx context.Context) (*google.Credentials, error) {
+	// Here we use a fake JSON credentials file, but you could also use
```

Comment:
Or this comment.

Reply:
I'm not sure what you're asking for here.

Reply:
I think you can remove the "Here we use a fake JSON credentials..." comment. Or reword it. It was written to be in the example code itself.

Reply:
Removed it.